ajax/libs/bootstrap-without-jquery/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "bootstrap-without-jquery",
+  "filename": "bootstrap3/bootstrap-without-jquery.min.js",
+  "version": "0.6.1",
+  "description": "Lightweight script to replace jQuery for simple interactions when using Twitter Bootstrap.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tagawa/bootstrap-without-jquery.git"
+  },
+  "keywords": [
+    "bootstrap",
+    "jquery"
+  ],
+  "author": "Daniel Davis",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tagawa/bootstrap-without-jquery.git",
+    "basePath": "",
+    "files": [
+      "bootstrap3/bootstrap-without-jquery*"
```

Comment:
I don't think you don't need "bootstrap2", main is just a form, the library is `bootstrap-without-jquery` not `bootstrap3-without-jquery`